scripts/scripts/testing/automations/batch/georgia.py

```diff
@@ -0,0 +1,23 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import datetime
+
+url = 'https://stopcov.ge'
+req = requests.get(url)
+soup = BeautifulSoup(req.text, 'html.parser')
+
+daily = data.find_all('span',attrs={'class':'quantity-numver'}[6])

```

Comment:
Variable `data` is not defined.

I tried substituting for `soup` but getting error:

```
daily = soup.find_all('span',attrs={'class':'quantity-numver'}[6])
KeyError: 6
```